test_src/sljitTest.c

```diff
@@ -11707,6 +11708,29 @@ static void test92(void)
 	sljit_emit_atomic_store(compiler, SLJIT_MOV_U16 | SLJIT_SET_ATOMIC_STORED, SLJIT_R2, SLJIT_R1, SLJIT_R0);
 	sljit_set_label(sljit_emit_jump(compiler, SLJIT_ATOMIC_NOT_STORED), label);
 
+#if !defined(SLJIT_CONFIG_ARM) || defined(__ARM_FEATURE_ATOMICS)
+	sljit_emit_op2(compiler, SLJIT_ADD, SLJIT_R1, 0, SLJIT_S0, 0, SLJIT_IMM, 32 * sizeof(sljit_sw));
```

Comment:
ARM now has compare and swap?  Sounds great! Compare and swap also used by x86 and s390x, so these tests should cover them. I would like to add a comment that these tests compare and swap operation specifically.